### PR TITLE
Boda Tweaks and Russian Drinks

### PR DESCRIPTION
**Pull request by klaasjared**
Cleans up that old 'Todo' that was tacked onto the Boda Vending Machine code way back when.

Adds three Russian drinks for the Boda Machine and subsequently adds them into the Boda machine.

_Drinks added:_
-Kvass
-Kompot
-Boda (sodawater)

Image of drinks below (cans in order of the list above).

russian_showcase

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
You seem to have included an unrelated arcade machine commit.

File: code/modules/food/food/cans.dm [hunk 1]
```diff
@@ -171,3 +171,36 @@
 	. = ..()
 	reagents.add_reagent("rootbeer", 30)
 
+//ADDITIONS BELOW THIS LINE MADE ON 03/01/2021
+
+/obj/item/weapon/reagent_containers/food/drinks/cans/kvass
+	name = "\improper Kvass"
+	desc = "A true slavic soda."
```
Comment (MistakeNot4892) [inline code/modules/food/food/cans.dm:178]: Slavic is a proper noun and should probably be capitalized.

File: code/modules/food/food/cans.dm [hunk 2]
```diff
@@ -171,3 +171,36 @@
...
+	description_fluff = "A classic slavic beverage which many Space Russians still enjoy to this day. Fun fact, it is actually considered a weak beer by Non-russians."
```
Comment (MistakeNot4892) [inline code/modules/food/food/cans.dm:179]: As above, also lowercase n on `Non`.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 3]
```diff
@@ -4092,4 +4092,32 @@
...
-	color = "#d169b2"
\ No newline at end of file
+	color = "#d169b2"
+
+//ADDITIONS BELOW THIS LINE MADE ON 02/28/2021
+
+/datum/reagent/drink/soda/kompot
+	name = "Kompot"
+	id = "kompot"
+	description = "A traditional eastern european beverage once used to preserve fruit in the 1980s"
```
Comment (MistakeNot4892) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:4102]: `Eastern European` I think. Not completely sure if `eastern` is part of the noun or not.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 4]
```diff
@@ -4092,4 +4092,32 @@
...
+	glass_desc = "A glass of refreshing kompot."
+	glass_special = list(DRINK_FIZZ)
+
+/datum/reagent/ethanol/kvass
+	name = "Kvass"
+	id = "kvass"
+	description = "A traditional fermented Slavic and Baltic beverage commonly made from rye bread."
+	taste_description = "a warm summer day at babushka's cabin"
+	color = "#b78315" // rgb: 183, 131, 21
+	strength = 95 //It's just soda to Russians
```
Comment (MistakeNot4892) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:4118]: Kvass is like 1-2% alcohol by volume compared to beer which is 4-6%. Seems like a bit of a meme to make it double the strength of beer.

**Comment by Atermonera** [comment]
`polaris.dme:762:error: unable to open "code/game/machinery/computer/arcade.dm".`

File: code/game/machinery/computer/arcade.dm [hunk 5]
```diff
@@ -1,1324 +0,0 @@
-/obj/machinery/computer/arcade
```
Comment (Atermonera) [inline code/game/machinery/computer/arcade.dm:1]: And now you've outright deleted the whole file as a committed change.

**Review by Atermonera — CHANGES_REQUESTED** [review]

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm (hunk 4 above)
Comment (MistakeNot4892) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:4118]: Discussed on Discord, higher is weaker in alcohol code. Thanks SS13.

**Comment by klaasjared** [comment]
Forgot to close this one - I'll be redoing it from scratch to fix the file errors

**Pull request closed without merging**